### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/agile-labor-categories#6]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)
## Results for https://pages.18f.gov/agile-labor-categories/
- **Error** The html element should have a lang or xml:lang attribute which describes the language of the document.
  - WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
  - `html`
  - `<html><head>
    <meta charset="utf-8...</html>`
- **Error** Anchor element found with a valid href attribute, but no link content has been supplied.
  - WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
  - `html > body > footer > div:nth-child(1) > div > p > a`
  - `<a href="https://github.com///edit//index.md"></a>`
### Summary

**Errors:** 2
